faust/transport/consumer.py

```diff
@@ -1122,6 +1122,9 @@ def _new_offset(self, tp: TP) -> Optional[int]:
             batch = next(consecutive_numbers(acked))
             # remove them from the list to clean up.
             acked[: len(batch)] = []
+            latest_commit_id = self._committed_offset.get(tp, -1)
```

Comment:
is there a unit test to validate this change?

Reply:
I don't think so. I think we should write one to make sure this doesn't break functionality.